src/EndpointConfig.php

```diff
@@ -55,6 +55,14 @@ public function parseError(stdClass $error): Error
         return Error::fromStdClass($error);
     }
 
+    /**
+     * Is it safe to display the errors from the endpoint to clients?
```

Comment:
```suggestion
     * Is it safe to display the error messages from the endpoint to clients?
```

Reply:
Errors can contain additional metadata in `extensions`, which is also subject to this setting.